### PR TITLE
Remove stopped transceivers from the set of transceivers

### LINKED ISSUE #2092: Remove stopped transceivers from getTransceivers()
Why keep them around when indices don't necessarily match the SDP anyway?

### PR DESCRIPTION
**Pull request by henbos**
Fixes #2092

***
<a href="https://pr-preview.s3.amazonaws.com/henbos/webrtc-pc/pull/2102.html" title="Last updated on Mar 7, 2019, 1:34 PM UTC (33412b7)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webrtc-pc/2102/9b5bb5b...henbos:33412b7.html" title="Last updated on Mar 7, 2019, 1:34 PM UTC (33412b7)">Diff</a>

### CODE REVIEW

**Comment by henbos** [comment]
AI: ask list if this is OK

**Comment by henbos** [comment]
Also stop filtering stopped transceivers if they are not present

**Comment by alvestrand** [comment]
More changes:
- Can remove the filter language from senders/receivers (if we remove at once)
- Need to also remove the transceiver when it's stopped because of port=0 in incoming SDP

**Comment by alvestrand** [comment]
Consensus of editors is that "remove immediately" makes sense.
Still need to address last week's points.

**Comment by henbos** [comment]
And the email thread "Proposal: Remove stopped transceivers to prevent resource exhaustion" has not had any responses (other than me and jan-ivar) in 6 days; removing "List discussion needed" label.

File: webrtc.html [hunk 1]
```diff
@@ -1882,9 +1882,8 @@ <h4>Set the RTCSessionDescription</h4>
                             </ol>
                           </li>
                           <li>
-                            <p>If the <a>media description</a> is rejected, and
-                            <var>transceiver</var> is not already stopped, <a>stop
-                            the RTCRtpTransceiver</a> <var>transceiver</var>.</p>
+                            <p>If the <a>media description</a> is rejected,
+                            <a>stop the RTCRtpTransceiver</a> <var>transceiver</var>.</p>
```
Comment (jan-ivar) [inline webrtc.html]: I claim we still need to check for already stopped to handle rollback for instance.

File: webrtc.html [hunk 2]
```diff
@@ -5166,12 +5175,6 @@ <h2>Methods</h2>
                       <code><a>RTCRtpTransceiver</a></code>, associated with
                       the sender, matches <var>kind</var>.</p>
                     </li>
-                    <li>
-                      <p>The transceiver is not <code>stopped</code>. More
-                      precisely, the <a>[[\Stopped]]</a> slot of the
-                      <code><a>RTCRtpTransceiver</a></code> associated with the
-                      sender is <code>false</code>.</p>
```
Comment (jan-ivar) [inline webrtc.html]: This was already redundant since *getSenders* was used. Nice catch!

**Comment by henbos** [comment]
Idea discussed: remove once stopped has been negotiated?

**Comment by henbos** [comment]
PR replaced by #2160

**Pull request closed without merging**